src/index.ts

```diff
@@ -2,44 +2,60 @@
 // this import must be called before the first import of tsyring
 import 'reflect-metadata';
 import { createServer } from 'http';
-import { createTerminus } from '@godaddy/terminus';
 import { Logger } from '@map-colonies/js-logger';
-import { container } from 'tsyringe';
-import config from 'config';
-import { DEFAULT_SERVER_PORT, SERVICES } from './common/constants';
-
-import { getApp } from './app';
+import { createTerminus } from '@godaddy/terminus';
+import { DependencyContainer } from 'tsyringe';
+import { JOB_CLEANER_FACTORY, LIVENESS_PROBE_FACTORY, DEFAULT_SERVER_PORT, SERVICES } from './common/constants';
 import { ShutdownHandler } from './common/shutdownHandler';
+import { ServerBuilder } from './httpServer/serverBuilder';
+import { registerExternalValues } from './containerConfig';
+import { JobsManager } from './manager/jobsManager';
+import { IConfig, IServerConfig } from './common/interfaces';
+
+let depContainer: DependencyContainer | undefined;
 
-interface IServerConfig {
-  port: string;
-}
+void registerExternalValues()
+  .then(async (container) => {
+    depContainer = container;
 
-const serverConfig = config.get<IServerConfig>('server');
-const port: number = parseInt(serverConfig.port) || DEFAULT_SERVER_PORT;
+    const livenessCheck = container.resolve<() => Promise<void>>(LIVENESS_PROBE_FACTORY);
+    container.resolve<void>(JOB_CLEANER_FACTORY);
 
-void getApp()
-  .then((app) => {
-    const logger = container.resolve<Logger>(SERVICES.LOGGER);
-    const stubHealthcheck = async (): Promise<void> => Promise.resolve();
+    const config = depContainer.resolve<IConfig>(SERVICES.CONFIG);
+    const serverConfig = config.get<IServerConfig>('server');
+    const port: number = parseInt(serverConfig.port) || DEFAULT_SERVER_PORT;
```

Comment:
change it so the value in the config is a number instead of parsing.

Reply:
done